test/CodeCoverage/codecoverage.bat

```diff
@@ -58,7 +58,7 @@ dotnet restore %REPOROOT%\test\Microsoft.SqlTools.ManagedBatchParser.Integration
 dotnet build %REPOROOT%\test\Microsoft.SqlTools.ManagedBatchParser.IntegrationTests\Microsoft.SqlTools.ManagedBatchParser.IntegrationTests.csproj %DOTNETCONFIG% 
 
 SET TEST_SERVER=localhost
-SET SQLTOOLSSERVICE_EXE=%REPOROOT%\src\Microsoft.SqlTools.ServiceLayer\bin\Integration\netcoreapp2.2\win7-x64\MicrosoftSqlToolsServiceLayer.exe
```

Comment:
Let's take the opportunity to move this into a var at the top of the file instead.